doc/Quick-Start-Guide.rst

```diff
@@ -8,7 +8,7 @@ Prerequisites
 *************
 The prerequisite that is necessary to install the Motr component is mentioned below.
 
-- CentOS-7 for x86_64 platform (ARM64 platform support work is in progress).
+- CentOS 7 or 8 on x86_64 or ARM64 (AArch64) platform.
```

Comment:
I know that CORTX-Motr may run on CentOS 8. But this is not officially fully tested/verified, and not announced publicly.

Reply:
Well, let's announce it this silent way, then. :)

It seems, the important question here is whether we will support and fix the reported bugs on CentOS 8. And I think the answer is - yes. Isn't it?

Let me know your opinion. If you don't agree - I will drop it.

Reply:
Ok, dropped for now.